packages/playwright-core/src/client/channelOwner.ts

```diff
@@ -176,6 +180,7 @@ export abstract class ChannelOwner<T extends channels.Channel = channels.Channel
     // Enclosing zone could have provided the apiName and wallTime.
     const expectZone = zones.zoneData<ExpectZone>('expectZone');
     const wallTime = expectZone ? expectZone.wallTime : Date.now();
```

Comment:
I think we can get rid of storing `wallTime` in the step zone and using it here now.

Reply:
This will break clients with old trace viewer opening the new trace as the wallTime will be out of sync for some actions now.